nexus/db-queries/src/db/datastore/inventory.rs

```diff
@@ -2731,6 +2746,12 @@ mod test {
                 .await
                 .unwrap();
             assert_eq!(0, count);
+            let count = schema::inv_dataset::dsl::inv_dataset
```

Comment:
This is the portion of the test where I *should* have looked for the inv_dataset being gone